cmake/onnxruntime_python.cmake

```diff
@@ -76,15 +58,7 @@ else()
   set(ONNXRUNTIME_SO_LINK_FLAG "-DEF:${ONNXRUNTIME_ROOT}/python/pybind.def")
 endif()
 
-if (onnxruntime_ENABLE_TRAINING)
```

Comment:
Can you keep this part unchanged?

Reply:
why? but providers depends on onnxruntime_training.

Reply:
let me try